pkg/keys/ecdsa_test.go

```diff
@@ -0,0 +1,44 @@
+package keys
+
+import (
+	"github.com/theupdateframework/go-tuf/data"
+	. "gopkg.in/check.v1"
+)
+
+type EcdsaSuite struct{}
+
+var _ = Suite(EcdsaSuite{})
+
+func (EcdsaSuite) TestSignVerify(c *C) {
```

Comment:
if you go with backwards compatibility, could you add a test with hard-coded deprecated key material, and make sure that properties like `PublicData()` and `Public()` and `Verify()` still work?